lib/logger/lib/logger/app.ex

```diff
@@ -34,6 +34,12 @@ defmodule Logger.App do
     primary_config = :logger.get_primary_config()
     :ok = :logger.set_primary_config(:level, default_level())
 
+    :ok =
+      Application.get_env(:logger, :metadata, [])
```

Comment:
```suggestion
      Application.fetch_env!(:logger, :metadata)
```

And set a default on `mix.exs`.

Reply:
Done!